src/rust/engine/src/scheduler.rs

```diff
@@ -468,6 +472,9 @@ impl Scheduler {
 
     let interval = ConsoleUI::render_interval();
     let deadline = request.timeout.map(|timeout| Instant::now() + timeout);
+    // TODO: Make this an option.
```

Comment:
This should likely be an optional frequency. And if it's been explicitly enabled, it might make sense to dump it at INFO level.

Reply:
presumably the option that enables this could accept a log level argument at which to dump the graph